app/lib/features/chat/pages/room_page.dart

```diff
@@ -194,8 +194,7 @@ class _ChatRoomConsumerState extends ConsumerState<ChatRoom> {
 
   void showMessageOptions(BuildContext context, types.Message message) {
     if (message is types.CustomMessage) {
-      if (message.metadata!.containsKey('eventType') &&
-          message.metadata!['eventType'] == 'm.room.redaction') {
+      if (message.metadata!['eventType'] == 'm.room.redaction') {
```

Comment:
This has probable chance of throwing null assertion if ```message.metadata``` is actually null.
```
if (message.metadata && message.metadata['eventType'] === 'm.room.redaction') {
    return;
}
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2140/commits/5f1691c0c57e555cc9471cd566259240c2a5f306